src/common/row_operations/row_heap_scatter.cpp

```diff
@@ -11,7 +11,7 @@ static void ComputeStringEntrySizes(UnifiedVectorFormat &vdata, idx_t entry_size
 	auto strings = (string_t *)vdata.data;
 	for (idx_t i = 0; i < ser_count; i++) {
 		auto idx = sel.get_index(i);
-		auto str_idx = vdata.sel->get_index(idx) + offset;
+		auto str_idx = vdata.sel->get_index(idx + offset);
```

Comment:
I initially pulled the `offset` out of the `get_index` because I was getting out of bounds on the selection vectors. Seems like that is no longer a problem, looking at the CI. LGTM!